opentracing-api/src/main/java/io/opentracing/ActiveSpanSource.java

```diff
@@ -39,12 +38,18 @@
     ActiveSpan activeSpan();
 
     /**
+     * @see #makeActive(Span, ActiveSpan.Observer)
```

Comment:
If we want to maintain backward compatibility of behaviour, then I think the javadoc comment for all of the methods that have an Observer variant should say that they represent the case of using the `AutoFinisher` observer, e.g. `SpanBuilder.startActive()`, `ActiveSpanSource.makeActive(Span)`, `Tracer.makeActive(Span)`.

We may also want to consider deprecating these methods, so that for the 1.0 API the Observer has to explicitly be specified, e.g. whether `null` or `AutoFinisher`.